recipes-extended/xen/xen-common.inc

```diff
@@ -116,4 +119,11 @@ PACKAGECONFIG[hvm] = "--with-system-seabios="/usr/share/firmware/bios.bin" --wit
 
 deltask stubs
 deltask deploy
+
+do_configure_append() {
+    if [ ! -e ${STAGING_INCDIR}/bits/long-double-32.h ]; then
+        cp ${STAGING_INCDIR}/bits/long-double-64.h ${STAGING_INCDIR}/bits/long-double-32.h
+        cp ${STAGING_INCDIR}/gnu/stubs-64.h ${STAGING_INCDIR}/gnu/stubs-32.h
```

Comment:
These headers will likely differ and renaming the 64 one to 32 will likely cause issues.
Why is that necessary though? I would expect `DISTRO_FEATURES` with `multiarch` to not require that.

The `meta-virtualization` recipe doesn't do that either.

Reply:
openembedded-core does for the gnu-efi recipe, just in the opposite direction.  It was also proposed for meta-virt in: https://lists.yoctoproject.org/pipermail/meta-virtualization/2017-August/002719.html though I don't know what became of the PR.

long-double-64.h is actually empty, and stubs-64.h is pretty minimal.  ```multiarch``` seems to do nothing if a single recipe builds 64 and 32 bit targets.